src/decode.rs

```diff
@@ -4425,28 +4425,23 @@ pub(crate) unsafe fn rav1d_decode_frame_init(
         f.lf.cdef_line[1][2] = offset.wrapping_add_signed(uv_stride_px * 6);
     }
 
-    let mut ptr =
-        f.lf.cdef_line_buf
-            .as_mut_ptr()
-            .add(32)
-            .offset(y_stride.abs() * f.sbh as isize * 4);
```

Comment:
The above bunch of additions to `offset` sums to this `32 + y_stride.abs() * f.sbh as isize * 4` here?

Reply:
Correct. This intermediate `ptr` value was being initialized to the value it would have had after the previous bunch of logic when the logic was working a pointer value instead of an offset. Now that both are using an offset, `offset` will have the correct value already at this point.